src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -12,6 +12,8 @@ interface CheckboxProps {
     | React.RefObject<HTMLInputElement>
     | null
     | undefined
+  tile?: boolean
+  labelDescription?: React.ReactNode
```

Comment:
I think this can be a `string`

Reply:
Resolved via https://github.com/trussworks/react-uswds/pull/1101#discussion_r612505761, `ReactNode` is good 👍